docs/data/sources/google-tag-manager-client.md

```diff
@@ -135,6 +149,57 @@ If the userId already available you can initialize the instance with a User ID.
     |`transport`| `TransportType.XHR` or `TransportType.SendBeacon` or `TransportType.Fetch`. Set the transport type. | `TransportType.Fetch` |
 - `Select a **GTM variable** from the list`. It's necessary to return an object containing the key-value pairs you wish to use for instance configuration. Ensure that the keys are part of the available configurations.
 
+##### Enable client side user agent enrichment
+
+Starting with version xxxxx, as of Aug, 2023, we have upgraded this template to use the Amplitude Browser SDK 2.0 for data collection. In Amplitude Browser SDK 2.0, we have deprecated client-side user agent parsing in favor of server-side user agent parsing. [More details](https://github.com/amplitude/Amplitude-TypeScript/tree/v1.x/packages/plugin-user-agent-enrichment-browser).
+
+To avoid breaking changes in chart analytics, we provide the option to enable client-side parsing. If you are a new user. We highly recommend adopting use server-side parsing, the new enrichment strategy offers more accurate result.
+
+##### Default Event Tracking
+
+This section include: 
+
+- Attribution Tracking
+- Page View Tracking
+- Sessions Tracking
+- Form Interaction Tracking
+- File downloads Tracking
+
+##### Attribution tracking
```

Comment:
I notice that the header level is quite deep here but can we make this and the following headers to header 6 with ###### as this should be nested in default event tracking in UI